docs/apps/lnw/es2k/es2k-lnw-underlay-lag.md

```diff
@@ -0,0 +1,378 @@
+<!--
+
+ Copyright 2024 Intel Corporation.
+
+ SPDX-License-Identifier: Apache-2.0
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at:
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+/
+- -->
+
+# Linux Networking with LAG (ES2K)
+
+This document explains how to run the Linux networking scenario with LAG (Link Aggregation Group) on ES2K.
+
+## Topology
+
+![Linux Networking Topology](es2k-lnw-topology.png)
+
+See [Linux Networking for ES2K](es2k-linux-networking.md)
+for more details on this feature.
+
+Prerequisites:
+
+- Download `hw-p4-programs` TAR file specific to the build and extract it to get `fxp-net_linux-networking-v2` p4 artifacts. Go through `Limitations` specified in `README` and bring up the setup accordingly.
+- Follow steps mentioned in [Deploying P4 Programs for E2100](/guides/es2k/deploying-p4-programs) for bringing up IPU with a custom P4 package.
+Modify `load_custom_pkg.sh` with following parameters for linux_networking package:
+
+```text
+    sed -i 's/sem_num_pages = 1;/sem_num_pages = 25;/g' $CP_INIT_CFG
+    sed -i 's/lem_num_pages = 1;/lem_num_pages = 10;/g' $CP_INIT_CFG
+    sed -i 's/acc_apf = 4;/acc_apf = 8;/g' $CP_INIT_CFG
+    sed -i 's/allow_change_mac_address = false;/allow_change_mac_address = true;/g' $CP_INIT_CFG
+```
+
+- Download `IPU_Documentation` TAR file specific to the build and refer to `Getting Started Guide` on how to install compatible `IDPF driver` on host. Once an IDPF driver is installed, bring up SRIOV VF by modifying the `sriov_numvfs` file present under one of the IDPF network devices. Example as below
+
+  ```bash
+  echo 1 > /sys/class/net/ens802f0/device/sriov_numvfs
+  ```
+
+Notes about topology:
+
+- VMs are spawned on top of each VF. Each VF will have a port representor in ACC. P4 runtime rules are configured to map VFs and their corresponding port representors.
+- Each physical port will have a port representor in ACC. P4 runtime rules are configured to map VFs and their corresponding port representors.
+- Each physical port will have a corresponding APF netdev on HOST. Create port representors in ACC for each HOST APF netdev. These APF netdevs on HOST will receive unknown traffic for applications to act on.
+- All port representors should be part of an OvS bridge. Based on topology, this OvS bridge will just perform bridging.
+- TEP termination on a bond interface which is used to enable underlay connectivity for VxLAN traffic.
```

Comment:
remove `which`
TEP termination on a bond interface is used to enable underlay connectivity for VxLAN traffic.